homeassistant/components/mpchc/media_player.py

```diff
@@ -106,13 +106,13 @@ def name(self):
     @property
     def state(self):
         """Return the state of the device."""
-        state = self._player_variables.get("statestring", None)
+        state = self._player_variables.get("state", None)
 
         if state is None:
             return STATE_OFF
-        if state == "playing":
+        if state == "2":
```

Comment:
Just to verify, `state` is returning the numbers as strings?

Reply:
`state` is coming from parsing a html page, and there is no conversion after that, so yes, it's a string.